pkgs/applications/editors/rstudio/default.nix

```diff
@@ -135,6 +136,13 @@ in
       ./pandoc-nix-path.patch
       ./use-system-quarto.patch
       ./ignore-etc-os-release.patch
+
+      (fetchpatch {
```

Comment:
`fetchpatch` is no longer in scope on master, can you rebase and add it back into the imports?

Reply:
thanks, added it back!